CHANGELOG.md

```diff
@@ -2,9 +2,22 @@
 
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
 
-## [0.18.2](https://github.com/puppetlabs/puppet_litmus/tree/0.18.2) (2020-05-28)
+## [0.18.3](https://github.com/puppetlabs/puppet_litmus/tree/0.18.3) (2020-06-09)
 
-[Full Changelog](https://github.com/puppetlabs/puppet_litmus/compare/v0.18.1...0.18.2)
+[Full Changelog](https://github.com/puppetlabs/puppet_litmus/compare/v0.18.2...0.18.3)
```

Comment:
Full Changelog should be prefix with v0.18.3

Reply:
oh, good spot!

since it is already broken on the previous link too, I'm assuming its a github-changelog-generator issue. Will follow-up there.